src/modules/0.0.20/aws_ecr/index.ts

```diff
@@ -105,6 +346,8 @@ class PublicRepositoryMapper extends MapperBase<PublicRepository> {
     delete: async (es: PublicRepository[], ctx: Context) => {
       const client = (await ctx.getAwsClient()) as AWS;
       for (const e of es) {
+        // need to delete images associated with this repository
+        await this.module.repositoryImages.deletePublicRepositoryImages(client.ecrPubClient, e);
```

Comment:
hmm I am not 100% sure about this one. we would be overriding the database functionality. The user should have deleted entries in the `image` table or done a delete query on `public_repository` with `CASCADE` and then the lazyLoader should converge on deleting the images and then the repository?